### ISSUE TITLE
Silent success (exit 0, "0 files") when project contains files with mdsvex `.svx` extension

### ISSUE THREAD

**Issue by KrwawyRosomak** [pheuter/svelte-check-rs#126]
## Summary

When a SvelteKit project registers a non-default Svelte extension via `svelte.config.js` (e.g. `.svx` from mdsvex), `svelte-check-rs` exits with code `0` and reports `0 errors and 0 warnings in 0 files` — even though valid `.ts` / `.svelte` files in the same `src/` tree contain real type errors. Passing `--ignore '**/*.svx'` makes the checker work correctly and surface those errors.

The failure is silent and exits 0, which is dangerous in CI — a "green" run actually checked nothing.

## Reproduction

```js
// svelte.config.js
import { vitePreprocess } from "@sveltejs/vite-plugin-svelte";
import { mdsvex } from "mdsvex";

export default {
  preprocess: [vitePreprocess(), mdsvex({ extensions: [".svx"] })],
  extensions: [".svelte", ".svx"],
};
```

```ts
// src/routes/blog/+page.ts
const _broken: number = "deliberate type error";
```

Plus at least one `.svx` file anywhere under `src/`.

## Actual

```
$ svelte-check-rs
====================================
svelte-check-rs found 0 errors and 0 warnings in 0 files
$ echo $?
0
```

## Expected

Either:

1. Check non-`.svx` files normally and skip `.svx` with a warning, e.g.
   ```
   warning: 9 files with unregistered extension (.svx) skipped
   svelte-check-rs found 1 error and 0 warnings in 1 file
   ```
2. Or abort with a non-zero exit code and a clear error explaining that the extension is not supported.

Anything is better than `exit 0` + "0 files".

## Workaround

```
svelte-check-rs --ignore '**/*.svx'
```

With the ignore flag the same project correctly reports the type error.

## Versions

- svelte-check-rs: 0.9.8
- svelte: 5.55.7
- mdsvex: 0.12.7
- Node: 24.15.0
- macOS: 26.5
